core/chains/ethereum2.go

```diff
@@ -0,0 +1,86 @@
+package chains
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type Ethereum2Response struct {
+	Page struct {
+		From int `json:"from"`
+		Size int `json:"size"`
+	} `json:"page"`
+	Total int    `json:"total"`
+	Next  string `json:"next"`
+	Data  []struct {
+		Id                        string  `json:"id"`
+		IdType                    string  `json:"idType"`
+		TimeWindow                string  `json:"timeWindow"`
+		ValidatorCount            int     `json:"validatorCount"`
+		AvgCorrectness            float64 `json:"avgCorrectness"`
+		AvgInclusionDelay         float64 `json:"avgInclusionDelay"`
+		AvgUptime                 float64 `json:"avgUptime"`
+		AvgValidatorEffectiveness float64 `json:"avgValidatorEffectiveness"`
+		ClientPercentages         []struct {
+			Client     string  `json:"client"`
+			Percentage float64 `json:"percentage"`
+		} `json:"clientPercentages"`
+		NetworkPenetration float64 `json:"networkPenetration"`
+	} `json:"data"`
+}
+
+type Ethereum2ErrorResponse struct {
+	Detail string `json:"detail"`
+}
+
+// https://api.rated.network/docs#/
+// the call used is for up to 100 entities but there are currently only 30
+// should update if operator count grows beyond 100
+//
+// voting powers were not included as the api does not provide this metric
+// and the ui does not display it
+func Ethereum2() (int, error) {
```

Comment:
suggest: changing `Ethereum` to just `Eth`

Reply:
done